docs/source/cpp/compute.rst

```diff
@@ -264,7 +264,9 @@ the input to a single output value.
   is always -1, regardless of whether there are nulls in the input.
 
 * \(5) For decimal inputs, the resulting decimal will have the same
-  precision and scale. The result is rounded away from zero.
+  scale and the maximum precision for its width. For instance, an array
```

Comment:
This is for `mean` right? And we didn't promote for `mean` in this PR.

Reply:
Good catch. Missed on the revision.